uinames.com/api/index.php

```diff
@@ -79,7 +79,7 @@ function generate_name ($database, $country = ANY, $language = ANY, $gender = AN
             }
         }
 
-        if (!$found) throw new Exception('No matching pool found');
+        if (!$found) throw new Exception('No matching country or language found');
```

Comment:
I'd prefer to keep calling this pool as it's not just country *or* language, it's any top level entry in the database, usually selected by just a country or a country and language combination, but never just a language. The code also commonly uses the word pool.